src/python/ensembl/io/genomio/gff3/process_gff3.py

```diff
@@ -325,6 +333,32 @@ def format_mobile_element(self, feat):
 
         return feat
 
+    def format_gene_segments(self, transcript: SeqFeature) -> SeqFeature:
+        """Returns the equivalent Ensembl biotype feature for gene segment transcript features.
+
+        Supported features: "C_gene_segment" and "V_gene_segment".
+
+        Args:
+            transcript: Gene segment transcript feature.
+
+        """
+        # Change V/C_gene_segment into a its corresponding transcript names
+        if transcript.type in ("C_gene_segment", "V_gene_segment"):
+            standard_name = transcript.qualifiers["standard_name"][0]
+            # Drop "_segment" from the transcript type
+            biotype = transcript.type[:-8]
```

Comment:
One line with a more explicit code:
```suggestion
            biotype = transcript.type.replace("_segment", "")
```